crawler/src/main/java/com/movie/recommender/crawler/controller/CrawlerController.java

```diff
@@ -0,0 +1,32 @@
+package com.movie.recommender.crawler.controller;
+
+import com.movie.recommender.crawler.service.MovieService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+@RequestMapping("/api/internal")
+@Slf4j
+public class CrawlerController {
+
+    private final MovieService movieService;
+
+    public CrawlerController(MovieService movieService) {
+        this.movieService = movieService;
+    }
+
+    @PostMapping("/crawl")
```

Comment:
path is fully defined at the class level
`@PostMapping` annotation doesn’t need an additional path